src/icon-button.tsx

```diff
@@ -124,7 +124,14 @@ const RawIconButton: React.FC<Props> = (props) => {
 
     if (props.onPress) {
         return (
-            <BaseTouchable {...commonProps} onPress={props.onPress} aria-label={props['aria-label']}>
+            <BaseTouchable
+                {...commonProps}
+                onPress={(event) => {
+                    event.stopPropagation();
```

Comment:
please, review other usages of `IconButton` in the repo, we may have some `stopPropagation` calls that we can remove now